plugins/auth-backend/src/service/router.ts

```diff
@@ -36,13 +35,61 @@ export async function createRouter(
   router.use(bodyParser.urlencoded({ extended: false }));
   router.use(bodyParser.json());
 
-  // configure all the providers
-  for (const providerConfig of providers) {
-    const { provider } = providerConfig;
-    const providerRouter = createAuthProviderRouter(providerConfig);
-    logger.info(`Configuring provider, ${provider}`);
-    router.use(`/${provider}`, providerRouter);
-  }
+  // TODO: read from app config
+  const config = {
+    backend: {
+      baseUrl: 'http://localhost:7000',
+    },
+    auth: {
+      providers: {
+        google: {
+          development: {
+            appOrigin: 'http://localhost:3000',
+            secure: false,
+            clientId: process.env.AUTH_GOOGLE_CLIENT_ID!,
+            clientSecret: process.env.AUTH_GOOGLE_CLIENT_SECRET!,
+          },
+          production: {
```

Comment:
remove this for now right? Or is it here for testing

Reply:
We added it for testing. Eventually, this will flow in from the config.